### PR TITLE
Added `where` to GraphQL query

### PR DESCRIPTION
**Pull request by devlife**
I noticed that the GraphQL query in the docs doesn't actually work.
The screenshot is the error I get when I use the `contentItemId` parameter.

image

And here is a successful query using `where`:

image

### CODE REVIEW

**Comment by devlife** [comment]
@sebastienros I added a screenshot of the error.

**Comment by sebastienros** [comment]
I trusted you ;) I am just surprised it was broken, or that it might have never worked.

**Comment by sebastienros** [comment]
My opinion is we might want to understand if it was broken, because it sounds like it used to work. So if it worked it should be fixed, and not just update the documentation.

File: src/docs/reference/core/Apis.GraphQL.Abstractions/README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ In this example, the `blog` is the type, and the `displayText` is the return val
 
 ```json
 {
-  blog(contentItemId: "4k5df0kadp9asy1n2ejzs1rz4r") {
+  blog(where {contentItemId: "4k5df0kadp9asy1n2ejzs1rz4r"}) {
```
Comment (jptissot) [inline src/docs/reference/core/Apis.GraphQL.Abstractions/README.md:19]:
```suggestion
  blog(where: {contentItemId: "4k5df0kadp9asy1n2ejzs1rz4r"}) {
```

**Pull request merged**